server/novaboot-shell

```diff
@@ -145,6 +195,15 @@ run_subcommand() {
 	    mkdir -p "$HOME/tftproot"
 	    cd "$HOME/tftproot"
 	    exec "$@";;
+	"rsync-nfsroot --server "*" . .")
+	    if ! is_nfs_enabled; then
+		die "NFS is not enabled"
+	    elif ! [ $# -eq 5 -o \( $# -eq 6 -a "$4" = '--log-format=X' \) ]; then
+ 		die "Unexpected rsync for nfsroot invocation: $*"
+	    fi
+	    mkdir -p "$HOME/nfsroot/${NB_USER:?}"
+	    cd "$HOME/nfsroot/${NB_USER:?}"
+	    exec "$@";;
```

Comment:
Chceck that `rsync` cannot overwrite files in subdirectories of `$HOME/nfsroot/${NB_USER:?}`.

Reply:
No, it cannot.